recipes/pysmithchart/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pysmithchart" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/p/pysmithchart/pysmithchart-{{ version }}.tar.gz
+  sha256: 9b10439a9b0e9883fac29308bfb5d78e18fca978197f52d5bc19efb436ef1cc6
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python {{ python_min }}
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - matplotlib-base
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pysmithchart
+  requires:
+    - python {{ python_min }}
+    - pip
+    - python {{ python_min }}
```

Comment:
Can you remove one of the duplicated `python {{ python_min }}` lines under `requires`? Then I'm happy to merge.

Reply:
thanks.  Missed that because I assume git merge worked.  Once again it messed me up.  Extra line removed.  thanks for the quick review.